torch/_export/serde/schema.py

```diff
@@ -185,12 +185,29 @@ class NamedArgument:
     arg: Argument
 
 
+@dataclass
+class NNModuleFrame:
+    key: str
+    path: str
+    ty: str
+
+
+@dataclass
+class NodeMeta:
+    stack_trace: Optional[str] = None
```

Comment:
@yiming0416 Should we leave these fields for later? Right now it's unclear to me what's the best way to schematize them.